cmd/controllers/main.go

```diff
@@ -79,31 +79,27 @@ func init() {
 }
 
 func RegisterSchemas(scheme *runtime.Scheme) error {
+	podAutoscalerEnabled := features.IsControllerEnabled(features.PodAutoscalerController)
+	modelAdapterEnabled := features.IsControllerEnabled(features.ModelAdapterController)
+	distributedInferenceEnabled := features.IsControllerEnabled(features.DistributedInferenceController)
+	kvCacheEnabled := features.IsControllerEnabled(features.KVCacheController)
+	stormServiceEnabled := features.IsControllerEnabled(features.StormServiceController)
 
-	if features.IsControllerEnabled(features.PodAutoscalerController) {
+	if podAutoscalerEnabled {
 		utilruntime.Must(autoscalingv1alpha1.AddToScheme(scheme))
 	}
 
-	if features.IsControllerEnabled(features.ModelAdapterController) {
+	if modelAdapterEnabled {
 		utilruntime.Must(modelv1alpha1.AddToScheme(scheme))
 	}
 
-	if features.IsControllerEnabled(features.DistributedInferenceController) {
-		utilruntime.Must(orchestrationv1alpha1.AddToScheme(scheme))
+	if distributedInferenceEnabled {
 		utilruntime.Must(rayclusterv1.AddToScheme(scheme))
 	}
 
-	if features.IsControllerEnabled(features.KVCacheController) {
-		utilruntime.Must(orchestrationv1alpha1.AddToScheme(scheme))
-	}
-
-	if features.IsControllerEnabled(features.StormServiceController) {
+	if distributedInferenceEnabled || kvCacheEnabled || stormServiceEnabled {
```

Comment:
do you know whether there's a way fine grain control the scheme. Currently, we have to register the group level.

Reply:
sure. let's keep group level at this moment